src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -222,6 +222,12 @@
   - name: --windows-admin-password
     type: string
     short-summary: Password to create on Windows node VMs.
+  - name: --enable-aad
+    type: bool
+    short-summary: Enable managed AAD feature for cluster.
+  - name: --aad-admin-group-object-ids
+    type: string
+    short-summary: Comma seperated list of aad group object IDs that will be set as cluster admin.
```

Comment:
Usually we use space separated for this case. And add `nargs='+'` for this argument in `_params.py`. There are so many examples in our code base, take [this ](https://github.com/Azure/azure-cli/blob/b5937aa49d572fe93e02bb2d30fa6b1cc575d224/src/azure-cli/azure/cli/command_modules/acr/_params.py#L144)for example.

If there is no special requirement for this parameter, I strongly suggest to follow above pattern. With that, we do not need to handle the parse logic ourselves.

Reply:
Thanks for reviewing the PR. In azure-cli-extension, we use the comma separated list for this argument. Is it OK that we use space separated for azure-cli and comma separated for azure-cli-extension?

Reply:
If they are the same parameter, it's better to keep them the same.